### PR TITLE
adjusted qsfs micro vm guide

### PR DESCRIPTION
**Pull request by Mik-TF**
* adjusted qsfs micro vm guide

Basically, added the mnemonics as environment variables (as well as cpu, size, memory and network), made a different file for environment variables, to have a simpler installation process. Also explained the minimum and expected shards parameters.

### CODE REVIEW

**Comment by Mik-TF** [comment]
@AhmedHanafy725 @ashraffouda @mohamedamer453 @xmonader @ramezsaeed @A-Harby 
Hi guys! Just a reminder if you could have a look at this PR. 

There is no rush per se, but I know there's a lot going on so I ping you here for the current PRs.

Thanks!

File: src/terraform/resources/terraform_qsfs_on_microvm.md [hunk 1]
```diff
@@ -1,12 +1,84 @@
...
+    * `Total CRU (Cores)`: 1
+    * `Free Public IP`: 2
+      * Note: if you want a public IPv4 address, it is recommended to set the parameter `FREE PUBLIC IP` to at least 2 to avoid false positives. This ensures that the shown 3Nodes have viable IP addresses.
+
+Once you've found a proper node, take node of its node ID. You will need to use this ID when creating the Terraform files.
+
+***
+
+## Create the Terraform Files
+
```
Comment (A-Harby) [inline src/terraform/resources/terraform_qsfs_on_microvm.md:60]: Can we also mention that all of these steps are already created in the Terraform repo and can be easily run by cloning it and simply running this [command](https://github.com/threefoldtech/terraform-provider-grid/tree/development#using-the-provider) ?

Comment (Mik-TF) [reply src/terraform/resources/terraform_qsfs_on_microvm.md:62]: Excellent idea @A-Harby !
I pushed a new commit with what you asked here. I also did a quick test to make sure it all works fine.

Thanks.

**Review by A-Harby — APPROVED** [review]
LGTM

**Pull request merged**